### PR TITLE
add test file

### PR DESCRIPTION
**Pull request by dungvq23**
Please help me to review

### CODE REVIEW

**Review by cathyduy — COMMENTED** [review]
Look ok

File: test.js [hunk 1]
```diff
@@ -0,0 +1 @@
+var a = 1;
```
Comment (cathyduy) [inline test.js:1]: should change to let keyword

**Review by vuonghuynhdn — COMMENTED** [review]
Work like a champ (y)

File: test.js (hunk 1 above)
Comment (dungvq23) [reply test.js:1]: Ok, I will change

**Pull request merged**